app/experimenter/static/js/scripts/edit-overview.js

```diff
@@ -0,0 +1,35 @@
+jQuery(function ($) {
```

Comment:
This whole thing is crappy but it was the only way I could find to reasonably hide/show required/optional fields while switching experiment types 🤷‍♀